_data/buildx/docker_buildx_create.yaml

```diff
@@ -33,6 +45,7 @@ options:
 - option: driver
   value_type: string
   description: 'Driver to use (available: [])'
```

Comment:
Should we use a backtick here 

![image](https://user-images.githubusercontent.com/47779042/112885904-2e077900-90c9-11eb-942c-b485b256e2ff.png)

Reply:
This description is coming from the generated code, and on the command-line we don't print backtics, so I'll need to have a look at this separately (we have the same issue on some of our other reference docs pages)